ruby/ql/lib/codeql/ruby/security/HardcodedDataInterpretedAsCodeQuery.qll

```diff
@@ -46,3 +44,34 @@ class Configuration extends DataFlow::Configuration {
     stateTo = FlowState::taint()
   }
 }
+
+/*
+ * We implement `DataFlow::ConfigSig` rather than
```

Comment:
Rather: We invoke `DataFlow::GlobalWithState` instead of `TaintTracking::GlobalWithState`